### ISSUE TITLE
Redirection Error 

### ISSUE THREAD

**Issue by mmarvel1** [johngodley/redirection#662]
``
I've just loaded the plugin and was looking at the 404 errors. So i went to create the redirect and then checked the box to remove the 404s.  It created the redirect but I get this error.  Not sure why.

Plugin: 3.0.1
WordPress: 4.9.2
PHP: 7.0.27
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: Unexpected token < in JSON at position 0
Action: redirection/v1/404?filterBy=url-exact&filter=%2Fwp-content%2Fuploads%2F2017%2F12%2Fwhstone-small.jpg DELETE
Code: 403 
Raw: <!DOCTYPE html>
<html style="height:100%">
<head><title> 403 Forbidden
</title></head>
<body style="color: #444; margin:0;font: normal 14px/20px Arial, Helvetica, sans-serif; height:100%; background-color: #fff;">
<div style="height:auto; min-height:100%; ">     <div style="text-align: center; width:800px; margin-left: -400px; position:absolute; top: 30%; left:50%;">
        <h1 style="margin:0; font-size:150px; line-height:150px; font-weight:bold;">403</h1>
<h2 style="margin-top:20px;font-size: 30px;">Forbidden
</h2>
<p>Access to this resource on the server is denied!</p>
</div></div></body></html>

```

**Comment by frankofno** [comment]
get the same error

**Comment by johngodley** [comment]
Can you try the 3.1 beta?

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by frankofno** [comment]
tried it right now. same error

**Comment by johngodley** [comment]
@frankofno can you post the error details you have now?

**Comment by frankofno** [comment]
Plugin: 3.1
WordPress: 4.9.1
PHP: 7.1.13
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_2) AppleWebKit/604.4.7 (KHTML, like Gecko) Version/11.0.2 Safari/604.4.7
REST API: http://www.cmphoto.at/index.php?rest_route=/

Error: JSON Parse error: Unrecognized token '<'
Action: redirection/v1/redirect&orderby=id&_wpnonce=27851f72d6 GET
Code: 403 Access Forbidden
Raw:  <html>
<head><title>403 Access Forbidden</title></head>
<body>
<div style="display: flex; align-items: center; justify-content: center; height: 70%;">
<div style="background-color: #eee; width: 70%; border: solid 3px #ddd; padding: 1.5em 3em 3em 3em; font-family: Arial, Helvetica, sans-serif;">
<h1>We're sorry, you are not allowed to proceed</h1>
<p>Server stopped processing your request. Your attempt to submit the form looks similar to automated requests from spam posting software.</p>
<p>If you believe you should be able to perform this request, please let us know.</p>
</div>
</div>
</body>
</html>


Error: JSON Parse error: Unrecognized token '<'
Action: redirection/v1/group&_wpnonce=27851f72d6 GET
Code: 403 Access Forbidden
Raw:  <html>
<head><title>403 Access Forbidden</title></head>
<body>
<div style="display: flex; align-items: center; justify-content: center; height: 70%;">
<div style="background-color: #eee; width: 70%; border: solid 3px #ddd; padding: 1.5em 3em 3em 3em; font-family: Arial, Helvetica, sans-serif;">
<h1>We're sorry, you are not allowed to proceed</h1>
<p>Server stopped processing your request. Your attempt to submit the form looks similar to automated requests from spam posting software.</p>
<p>If you believe you should be able to perform this request, please let us know.</p>
</div>
</div>
</body>
</html>

**Comment by johngodley** [comment]
@frankofno if you go to the Redirection support page, does the 'plugin status' show all good, or is anything listed as a warning? If so, could you press on the 'magix fixer' button?

**Comment by frankofno** [comment]
pressed the button, since the REST API is showing PROBLEM in orange.

WordPress REST API | Problem REST API returns an error code (403 - http://www.cmphoto.at/index.php?rest_route=/)
-- | --
Good All tables present
Good Valid groups detected
Good All redirects have a valid group
Good Post monitor group is valid

**Comment by johngodley** [comment]
@frankofno I updated the beta to handle an extra situation, and I think it may be enough to fix this for you. You'll need to click the magic fixer again. Hopefully the REST API section should state it is using wp-admin/admin-ajax.php instead

If that still doesnt fix things then I think you are running some plugin on your site that is blocking the requests. If that is something you are doing could you list what the plugin(s) are?

**Comment by frankofno** [comment]
still an error on the REST API

Plugin: 3.1
WordPress: 4.9.1
PHP: 7.1.13
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/604.5.6 (KHTML, like Gecko) Version/11.0.3 Safari/604.5.6
REST API: http://www.cmphoto.at/wp-json/

Error: JSON Parse error: Unrecognized token '<'
Action: redirection/v1/setting GET
Code: 403 Access Forbidden
Raw:  <html>
<head><title>403 Access Forbidden</title></head>
<body>
<div style="display: flex; align-items: center; justify-content: center; height: 70%;">
<div style="background-color: #eee; width: 70%; border: solid 3px #ddd; padding: 1.5em 3em 3em 3em; font-family: Arial, Helvetica, sans-serif;">
<h1>We're sorry, you are not allowed to proceed</h1>
<p>Server stopped processing your request. Your attempt to submit the form looks similar to automated requests from spam posting software.</p>
<p>If you believe you should be able to perform this request, please let us know.</p>
</div>
</div>
</body>
</html>


Error: JSON Parse error: Unrecognized token '<'
Action: redirection/v1/plugin GET
Code: 403 Access Forbidden
Raw:  <html>
<head><title>403 Access Forbidden</title></head>
<body>
<div style="display: flex; align-items: center; justify-content: center; height: 70%;">
<div style="background-color: #eee; width: 70%; border: solid 3px #ddd; padding: 1.5em 3em 3em 3em; font-family: Arial, Helvetica, sans-serif;">
<h1>We're sorry, you are not allowed to proceed</h1>
<p>Server stopped processing your request. Your attempt to submit the form looks similar to automated requests from spam posting software.</p>
<p>If you believe you should be able to perform this request, please let us know.</p>
</div>
</div>
</body>
</html>
